src/core/ext/client_config/lb_policy_factory.c

```diff
@@ -46,6 +47,23 @@ grpc_lb_addresses* grpc_lb_addresses_create(size_t num_addresses) {
   return addresses;
 }
 
+grpc_lb_addresses* grpc_lb_addresses_copy(grpc_lb_addresses* addresses,
+                                          void* (*user_data_copy)(void*)) {
+  grpc_lb_addresses* new = grpc_lb_addresses_create(addresses->num_addresses);
```

Comment:
not sure, but using `new` as a variable name may wreak havoc when combining this with C++, being a reserved keyword. `s/new/new_lb_addr/`?